dynaml/reference.go

```diff
@@ -23,17 +26,22 @@ func (e ReferenceExpr) Evaluate(binding Binding) (yaml.Node, bool) {
 			step, ok = binding.FindReference(e.Path[:i+1])
 		}
 
+		debug.Debug("  %d: %v %+v\n",i,ok,step)
 		if !ok {
-			return nil, false
+			info.Issue="'"+strings.Join(e.Path,".")+"'"+" not found"
```

Comment:
This is a bit nit-picky, but we're going to ask anyway: this string concatenation is a bit hairy. Can you please modify this to use`fmt.Sprintf()`? We believe `info.Issue = fmt.Sprintf("'%s' not found", strings.Join(e.Path, "."))` is a clearer way of showing the reader exactly how the issue string will be formatted.